tutorial/7 - advanced.test.ts

```diff
@@ -286,16 +289,15 @@ describe.skip('advanced', () => {
              * We will set `secondProp`, will this affect `firstProp$`?
              */
             myMap$.swap(map => map.set('secondProp', 'new value'));
-            expect(reactPropSpy).to.have.callCount(__YOUR_TURN__)   // how many times was the spy called?
-                .and.calledWith(__YOUR_TURN__);                     // and with what value?
```

Comment:
This assertion is missing in the updated version

Reply:
Yes, I removed as the callCount is 0, I found it confusing to have a calledWith call as it should be removed.